src/services/AuthAPI.ts

```diff
@@ -32,4 +32,27 @@ async function loginUser(
   }
 }
 
-export { loginUser };
+async function logoutUser(): Promise<
+  | {
+      ok: true;
```

Comment:
You should add `message: string` here as well.

Reply:
We can remove some duplicate code. Since both types have message, we can remove the OR and ahve just one type

```
ok: boolean;
message: string;
```